test/env.js

```diff
@@ -4,6 +4,29 @@ global.requestAnimationFrame = jest.fn().mockImplementation(() => {
   throw new Error('requestAnimationFrame is not supported in Node')
 })
 
+const allListeners = {}
+
+global.matchMedia = function matchMedia(media) {
+  allListeners[media] = allListeners[media] || []
+
+  return {
+    media,
+    matches: false,
+    addListener: listener => allListeners[media].push(listener),
+    removeListener: listener => {
+      let i = allListeners[media].length - 1
+
+      while (i >= 0) {
+        if (allListeners[media][i] === listener) {
+          allListeners[media].splice(i, 1)
+        }
+        i -= 1
+      }
+    },
+    onchange: () => allListeners[media].forEach(listener => listener({})),
```

Comment:
nit: onChange

Reply:
this is part of the MediaQueryList spec https://drafts.csswg.org/cssom-view/#dom-mediaquerylist-onchange